modules/@apostrophecms/piece-type/index.js

```diff
@@ -388,20 +395,61 @@ module.exports = {
       },
       'apostrophe:modulesRegistered': {
         composeBatchOperations() {
-          self.batchOperations = Object.keys(self.batchOperations).map(key => ({
-            action: key,
-            ...self.batchOperations[key]
-          })).filter(batchOperation => {
-            // If a `requiredField` is registered, only include the operation
-            // if that field is present on the schema.
-            if (batchOperation.requiredField && !_.find(self.schema, {
-              name: batchOperation.requiredField
-            })) {
-              return false;
+          const groupedOperations = Object.entries(self.batchOperations)
+            .reduce((acc, [ opeName, properties ]) => {
```

Comment:
A smaller thing, but using `ope` to abbreviate `operation` is confusing to me. A more common abbreviation for the word is `op`. There are a few variables this applies to below.

Reply:
Ok, changed.